src/cmd/nozzle/app/nozzle.go

```diff
@@ -203,4 +216,4 @@ func (app *NozzleApp) startProfilingServer() {
 	app.profilingMutex.Lock()
 	app.profilingListener = metrics.StartProfilingServer(app.cfg.ProfilingAddr, app.log)
 	app.profilingMutex.Unlock()
-}
```

Comment:
remove empty line

Reply:
git is not allowing me to remove this line. When I remove the line, git is not recognised as a change